packages/default/scss/drawer/_theme.scss

```diff
@@ -0,0 +1,62 @@
+@include exports("drawer/theme") {
+    .k-drawer-container {
+        background-color: $drawer-bg;
+        color: $drawer-text;
+        border-color: $drawer-border;
+    }
+
+    .k-drawer-container,
+    .k-drawer-container .k-drawer-items {
```

Comment:
If you decide to move the relevant styles in the `_layout`, apply the same logic here.